### DISCUSSION TITLE
AWS Lambda Permissions Issue

### DISCUSSION THREAD

**Discussion by geoffhigginbottom** [open-telemetry/opentelemetry-lambda#11] [Q&A]
Been trying to test this but it keeps failing with the following error when I try and attach the layer

not authorized to perform: lambda:GetLayerVersion on resource: arn:aws:lambda:eu-west-3:297975325230:layer:opentelemetry-lambda-extension:8

I am using an administrator account with "full access" but it still fails.

Also tried it via the UI and get the same message "You are not authorized to perform: lambda:GetLayerVersion."

Looks like we are not able to query the version of the layer

**Comment by AbhaySingh31** [comment]
Hi @geoffhigginbottom did you find a solution for this ?  i'm facing same problem

**Comment by tylerbenson** [comment]
Please try again with any of the new layer releases.
